plotly_resampler/aggregation/aggregators.py

```diff
@@ -90,7 +91,8 @@ def _arg_downsample(
         **_,
     ) -> np.ndarray:
         # Use the Core interface to perform the downsampling
-        return LTTB_core.downsample(x, y, n_out)
+        # return LTTB_core.downsample(x, y, n_out)
+        return LTTBDownsampler().downsample(x, y, n_out=n_out)
 
 
 class MinMaxOverlapAggregator(DataPointSelector):
```

Comment:
Do we remove this one? - I do not really see the point of keeping this one...

Reply:
ineed, I would remove the `LTTB_core`